django_forest/resources/utils/csv.py

```diff
@@ -16,13 +16,12 @@ def fill_csv_relationships(self, res, record, data, params):
         for name, value in record['relationships'].items():
             related_res = self.get_related_res(data, value)
             if related_res:
-                res[name] = related_res['attributes'][params[f'fields[{name}]']]
+                res[name] = related_res['id']
         return res
 
     def fill_csv(self, data, writer, params):
         for record in data['data']:
             res = record['attributes']
-            res['id'] = record['id']
```

Comment:
Removing this line because:
1. It throws an error when there is no column named "id" in the table.
2. It seems to be rendundent because the line above (`res = record['attributes']`) already sets the values for all non-foreign-key columns.